### PR TITLE
Enable CMAQ scavenging and wet removal in resolved clouds

### PR DESCRIPTION
**Pull request by rmontuoro**
This PR enables CMAQ-provided simplistic scavenging and wet removal in resolved clouds to address issue #23.

A ufs-weather-model [branch](https://github.com/rmontuoro/ufs-weather-model/tree/feature/aqm-rescld-wdep) is available for testing purposes:
```
 git clone -b feature/aqm-rescld-wdep --recursive https://github.com/rmontuoro/ufs-weather-model
```

**Note:** This branch also includes [changes to fv3atm](https://github.com/rmontuoro/fv3atm/tree/feature/enable-aqm-aod) enabling AOD output. A PR need to be created to merge such changes into the fv3atm develop branch.

Depends on NOAA-EMC/fv3atm/pull/593
Part of ufs-community/ufs-weather-model/pull/1456

### CODE REVIEW

**Comment by JianpingHuang-NOAA** [comment]
What is the difference between PRs#23 and #24?

**Comment by bbakernoaa** [comment]
@JianpingHuang-NOAA I don't see a PR#23

**Comment by JianpingHuang-NOAA** [comment]
A typo. It is issue 23#

**Comment by BrianCurtis-NOAA** [comment]
@rmontuoro It looks like baselines are changing, is this expected?

```
baseline dir = /scratch1/NCEPDEV/nems/emc.nemspara/RT/NEMSfv3gfs/develop-20221007/INTEL/regional_atmaq
working dir  = /scratch1/NCEPDEV/stmp2/Brian.Curtis/FV3_RT/rt_11339/regional_atmaq
Checking test 001 regional_atmaq results ....
 Comparing sfcf000.nc ............ALT CHECK......NOT OK
 Comparing sfcf003.nc ............ALT CHECK......NOT OK
 Comparing sfcf006.nc ............ALT CHECK......NOT OK
 Comparing atmf000.nc ............ALT CHECK......NOT OK
 Comparing atmf003.nc ............ALT CHECK......NOT OK
 Comparing atmf006.nc ............ALT CHECK......NOT OK
 Comparing RESTART/coupler.res .........OK
 Comparing RESTART/fv_core.res.nc .........OK
 Comparing RESTART/fv_core.res.tile1.nc .........OK
 Comparing RESTART/fv_srf_wnd.res.tile1.nc .........OK
 Comparing RESTART/fv_tracer.res.tile1.nc ............ALT CHECK......NOT OK
 Comparing RESTART/phy_data.nc .........OK
 Comparing RESTART/sfc_data.nc .........OK
```

**Comment by rmontuoro** [comment]
@BrianCurtis-NOAA - Yes. The AQM baseline (tracers) is expected to change since new physical processes are now enabled.

**Comment by BrianCurtis-NOAA** [comment]
@rmontuoro Do you mind if i merge in ytangnoaa's pt-source changes into this for getting 2 changes into UFSWM?

**Comment by rmontuoro** [comment]
@BrianCurtis-NOAA - I would like to review PR #18 before merging.

**Comment by BrianCurtis-NOAA** [comment]
> @BrianCurtis-NOAA - I would like to review PR #18 before merging.

@rmontuoro I will start testing with this PR w/o merging in AQM PR #18 at 2:00 if I don't hear back from you about reviewing AQM PR #18, and will work on AQM PR #18 tomorrow.

**Comment by BrianCurtis-NOAA** [comment]
@JianpingHuang-NOAA has asked me to hold on this PR as he talks with @rmontuoro about some issues.

**Comment by rmontuoro** [comment]
@BrianCurtis-NOAA - The latest updates reproduce bit-per-bit the results from @JianpingHuang-NOAA's original changes in the `regional_atmaq` regression test.

**Comment by BrianCurtis-NOAA** [comment]
Great, I may look to get this into tomorrow's UFSWM pr. As long as everything goes smoothly with the current UFSWM PR.

**Comment by rmontuoro** [comment]
@BrianCurtis-NOAA - PR #32 was merged into this PR.

**Comment by BrianCurtis-NOAA** [comment]
@bbakernoaa @JianpingHuang-NOAA Please provide a review of the code as we will merge this PR ASAP.

File: examples/aqm.rc [hunk 1]
```diff
@@ -35,9 +35,15 @@ omi_data:    /scratch1/NCEPDEV/nems/Raffaele.Montuoro/dev/aqm/epa/data/omi_cmaq_
...
-# Run aerosol module
+# Run options:
+#
+# - enable aerosol module
 #
 run_aerosol: true
+#
+# - enable wet deposition processes in resolved-scale clouds
+#
+run_rescld: true
```
Comment (bbakernoaa) [inline examples/aqm.rc:46]: Like the option here

**Review by bbakernoaa — APPROVED** [review]

**Review by JianpingHuang-NOAA — APPROVED** [review]

**Pull request merged**